src/validator.test.js

```diff
@@ -0,0 +1,11 @@
+const { validateEverything } = require("./validator");
+const validTestData = require("../fixtures/valid-survey.json");
+
+describe("Validation Script", () => {
+  it("Wont throw an error if there aren't any", () => {
+    const errors = {
+      errors: []
+    };
+    expect(() => validateEverything(validTestData, errors)).not.toThrow();
+  });
```

Comment:
Could probably do with a test that asserts it does throw an error when there is one?

Reply:
Yeah, that would be valuable. I'll add that in. I think that I didn't originally because I was testing each individual validation separately, but on second thoughts that wouldn't be testing that it's integrated properly with the rest of the validator.